minimal/docker-compose.yml

```diff
@@ -53,4 +53,13 @@ services:
     volumes_from:
       - {{ project_name }}_web
     ports:
-      - "0.0.0.0:80:80"
\ No newline at end of file
+      - "0.0.0.0:80:80"
+
+  {{ project_name }}_certbot:
+    restart: always
+    build: ./compose/certbot/production
+    depends_on:
+      - {{ project_name }}_web
+    volumes_from:
```

Comment:
@pslobo but i wanted to change nginx to become:
```
  smal_nginx:
    restart: always
    build: ./compose/nginx/staging
    depends_on:
      - smal_web
    volumes:
      - ./media:/code/media
      - ./staticfiles:/code/staticfiles
    ports:
      - "0.0.0.0:8008:80"
```
You are saying that is not enough for cerbot? This will give you access to the staticfiles and media files if certbot only writes to the staticfiles folder this should work shouldn't it??

If not can you illustrate with some code here as a comment your second idea?